extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,126 @@
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**transmissionFactor** | `number` | The base percentage of light that is transmitted through the surface.| No, default:`1.0`|
+|**transmissionTexture** | [`textureInfo`] | A greyscale texture that defines the transmission percentage of the surface. This will be multiplied by transmissionFactor. | No |
+
+### transmissionFactor 
+
+The amount of light that is transmitted by the surface rather than diffusely re-emitted. This is a percentage of all the light that penetrates a surface (i.e. isn’t specularly reflected) rather than a percentage of the total light that hits a surface. A value of 1.0 means that 100% of the light that penetrates the surface is transmitted through. This is the default.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>Defaulting to 1.0 allows making a material transparent by simple defining the extension.</em></figcaption>
+</figure>
+
+### transmissionTexture 
+
+A greyscale texture that defines the amount of light that is transmitted by the surface rather than absorbed and re-emitted. A value of 1.0 means that 100% of the light that penetrates the surface (i.e. isn’t specularly reflected) is transmitted through. The value is linear and is multiplied by the transmissionFactor to determine the total transmission value. Note that use of transmission values does not disallow usage of alpha coverage (via `baseColor.a`).
+
+<figure>
+  <img src="./figures/TransmissionTexture.png"/>
+<figcaption><em>Controlling transmission amount with transmissionTexture.</em></figcaption>
+</figure>
+
+## Implementing Transmission ##
+
+Modeling transmission is relatively straightforward. From the [glTF BRDF](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#appendix-b-brdf-implementation), we have:
+
+*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub>
+*f*<sub>*diffuse*</sub> = (1 - *F*) * *diffuse*
+*f*<sub>*specular*</sub> = *F* * *G* * *D* / (4 * dot(*N*, *L*) * dot(*N*, *V*))
+, where *F* is the Surface Reflection Ratio.
+
+*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub>
+
+Optical transparency does not require any changes whatsoever to the specular term so we only want to modify the diffuse term to account for transmitted light as well. We can do this by blending *f*<sub>*diffuse*</sub> with the transmitted light, **L**<sub>transmitted</sub>, based on the `transmission` values.
+
+*f* = mix(**f**<sub>*diffuse*</sub>, **L**<sub>transmitted</sub> * `baseColor`, **T**) + *f*<sub>*specular*</sub>
+
+Where **L**<sub>transmitted</sub> is the colour of light coming through the material, towards the eye. It is tinted (multiplied) by the `baseColor`, and already has refraction taken into account (as discussed below). *T* is the transmission value defined by this extension.
```

Comment:
How is **L**<sub>transmitted</sub> computed? I guess it includes (1-F)?

Reply:
I don't have a description of **L**<sub>transmitted</sub> in here yet but, yes, it would have to include 1-F. The IOR for that calculation would come from the base material (or an extension that overrides the base F0 of 0.04).

Reply:
I've cleaned up the math a bit here.